app/src/test/java/org/oppia/app/profile/ProfileChooserFragmentLocalTest.kt

```diff
@@ -0,0 +1,134 @@
+@LooperMode(LooperMode.Mode.PAUSED)
+@Config(
+  application = ProfileChooserFragmentLocalTest.TestApplication::class,
+  qualifiers = "port-xxhdpi"
+)
+class ProfileChooserFragmentLocalTest {
+
+  @Inject
+  lateinit var fakeEventLogger: FakeEventLogger
+
+  @Before
+  fun setUp() {
+    Intents.init()
+    setUpTestApplicationComponent()
+  }
+
+  @After
+  fun tearDown() {
+    Intents.release()
+  }
+
+  @Test
+  fun testProfileChooser_onLaunch_logsEvent() {
+    launch<ProfileActivity>(createProfileActivityIntent()).use {
+      val event = fakeEventLogger.getMostRecentEvent()
+
+      assertThat(event.priority).isEqualTo(Priority.ESSENTIAL)
+      assertThat(event.actionName).isEqualTo(EventAction.OPEN_PROFILE)
+      assertThat(event.context.activityContextCase).isEqualTo(ACTIVITYCONTEXT_NOT_SET)
+    }
+  }
+
+  private fun createProfileActivityIntent(): Intent {
+    return ProfileActivity.createProfileActivity(ApplicationProvider.getApplicationContext())
+  }
+
+  private fun setUpTestApplicationComponent() {
+    ApplicationProvider.getApplicationContext<TestApplication>().inject(this)
+  }
+
+  @Module
+  class TestModule {
+    // Do not use caching to ensure URLs are always used as the main data source when loading audio.
+    @Provides
+    @CacheAssetsLocally
+    fun provideCacheAssetsLocally(): Boolean = false
+  }
+
+  // TODO(#59): Figure out a way to reuse modules instead of needing to re-declare them.
+  @Singleton
+  @Component(
+    modules = [
+      TestModule::class, TestDispatcherModule::class, ApplicationModule::class, NetworkModule::class,
+      LoggerModule::class, ContinueModule::class, FractionInputModule::class,
+      ItemSelectionInputModule::class, MultipleChoiceInputModule::class,
+      NumberWithUnitsRuleModule::class, NumericInputRuleModule::class, TextInputRuleModule::class,
+      DragDropSortInputModule::class, InteractionsModule::class, GcsResourceModule::class,
+      GlideImageLoaderModule::class, ImageParsingModule::class, HtmlParserEntityTypeModule::class,
+      QuestionModule::class, TestLogReportingModule::class
+    ]
+  )
+  interface TestApplicationComponent : ApplicationComponent {
+    @Component.Builder
+    interface Builder : ApplicationComponent.Builder
+
+    fun inject(profileChooserFragmentLocalTest: ProfileChooserFragmentLocalTest)
+  }
+
+  class TestApplication : Application(), ActivityComponentFactory {
+    private val component: TestApplicationComponent by lazy {
+      DaggerProfileChooserFragmentLocalTest_TestApplicationComponent.builder()
+        .setApplication(this)
+        .build() as TestApplicationComponent
+    }
+
+    fun inject(profileChooserFragmentLocalTest: ProfileChooserFragmentLocalTest) {
+      component.inject(profileChooserFragmentLocalTest)
+    }
+
+    override fun createActivityComponent(activity: AppCompatActivity): ActivityComponent {
+      return component.getActivityComponentBuilderProvider().get().setActivity(activity).build()
+    }
+  }
+}
```

Comment:
Add new line at EOF. Maybe run `ktlint --android -F app/src/test/java/org/oppia/app/profile/ProfileChooserFragmentLocalTest.kt` once on this file (As well as organize imports and reformat code  commands of Android Studio on this file)

Reply:
Done.